flexget/plugins/urlrewrite_nnmclub.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+from flexget.utils import requests
+
+from bs4 import BeautifulSoup
+
+from flexget import plugin
+from flexget.event import event
+
+log = logging.getLogger('nnm-club')
+
+
+class UrlRewriteNnmClub(object):
+    """Nnm-club.me urlrewriter."""
+
+    def url_rewritable(self, task, entry):
+        return entry['url'].startswith('http://nnm-club.me/forum/viewtopic.php?t=')
+
+    def url_rewrite(self, task, entry):
+        initUrl = entry['url']
+        html = requests.get(initUrl).content
```

Comment:
We could actually just use `task.requests` here, that means plugins like `modify_headers` will work with this plugin's requests.